src/Resources/app/administration/src/mixin/swag-paypal-merchant-information.mixin.ts

```diff
@@ -2,7 +2,7 @@ import type * as PayPal from 'src/types';
 
 export default Shopware.Mixin.register('swag-paypal-merchant-information', Shopware.Component.wrapComponentConfig({
     inject: [
-        'SwagPayPalApiCredentialsService',
+        'SwagPayPalSettingsService',
     ],
```

Comment:
isnt't that a break?

Reply:
No, the functionality remains the same, except for the validateCredentials route - as it's still behind the feature flag and wasn't released, it should be fine anyway